sematic/ui/packages/common/src/pages/RunDetails/index.tsx

```diff
@@ -24,7 +26,7 @@ const RunDetails = (props: RunDetailsProps) => {
 
     const [selectedRunIdHash, setSelectedRunId] = useAtom(selectedRunHashAtom);
     const [selectedPanel, setSelectedPanel] = useAtom(selectedPanelAtom);
-
+    const [, setSelectRun ] = useAtom(selectedRunAtom);
```

Comment:
Unfortunately, as a component external to RunDetails page, the `addon-grafana` cannot access the context value created here by `RunDetailsSelectionContext.Provider`. As a workaround, I am sharing this information through a Jotai Atom, which can be accessed globally within the React tree.